<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISF Conductance Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
    <item>
     <widget class="QWidget" name="plotWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="plotTab1">
          <attribute name="title">
           <string>Input 1</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QWidget" name="plotWidget1" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="PlotWidget" name="sourcePlot1" native="true"/>
              </item>
              <item>
               <widget class="PlotWidget" name="sinkPlot1" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="plotTab2">
          <attribute name="title">
           <string>Input 2</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QWidget" name="plotWidget2" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="PlotWidget" name="sourcePlot2" native="true"/>
              </item>
              <item>
               <widget class="PlotWidget" name="sinkPlot2" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QScrollBar" name="plotScrollBar">
         <property name="maximum">
          <number>0</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>1000</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="invertedControls">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="toolWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="setupWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="setupLabel">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Setup</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QStackedWidget" name="setupWidgetStack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="SetupWidget" name="setupWidget1"/>
            <widget class="SetupWidget" name="setupWidget2"/>
            <widget class="SetupWidget" name="setupWidget3"/>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="analysisWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QLabel" name="analysisLabel">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Analysis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="analysisGroup1">
            <property name="title">
             <string>Input 1</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="AnalysisWidget" name="analysisWidget1" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="analysisGroup2">
            <property name="title">
             <string>Input 2</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="AnalysisWidget" name="analysisWidget2" native="true"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionResumeCapture"/>
   <addaction name="actionPauseCapture"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionResetZoom"/>
   <addaction name="separator"/>
   <addaction name="actionAutoScale"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/connect.svg</normaloff>:/res/connect.svg</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/disconnect.svg</normaloff>:/res/disconnect.svg</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionResumeCapture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/start.svg</normaloff>:/res/start.svg</iconset>
   </property>
   <property name="text">
    <string>Resume Capture</string>
   </property>
   <property name="toolTip">
    <string>Resume Capture</string>
   </property>
  </action>
  <action name="actionPauseCapture">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/pause.svg</normaloff>:/res/pause.svg</iconset>
   </property>
   <property name="text">
    <string>Pause Capture</string>
   </property>
   <property name="toolTip">
    <string>Pause Capture</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/zoom-in-x.svg</normaloff>:/res/zoom-in-x.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/zoom-out-x.svg</normaloff>:/res/zoom-out-x.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionResetZoom">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/zoom-original.svg</normaloff>:/res/zoom-original.svg</iconset>
   </property>
   <property name="text">
    <string>Reset Zoom</string>
   </property>
   <property name="toolTip">
    <string>Reset Zoom</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/about.svg</normaloff>:/res/about.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAutoScale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ConductanceViewer.qrc">
     <normaloff>:/res/autoscale.svg</normaloff>:/res/autoscale.svg</iconset>
   </property>
   <property name="text">
    <string>Auto-Scale</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SetupWidget</class>
   <extends>QWidget</extends>
   <header>setupwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>plotwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnalysisWidget</class>
   <extends>QWidget</extends>
   <header>analysiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ConductanceViewer.qrc"/>
 </resources>
 <connections/>
</ui>
